python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/mpls.j2

```diff
@@ -35,3 +35,120 @@ mpls icmp fragmentation-needed tunneling
 mpls icmp ttl-exceeded tunneling
 {%     endif %}
 {% endif %}
+{% if mpls.rsvp is arista.avd.defined %}
+!
+mpls rsvp
+{%     if mpls.rsvp.refresh is arista.avd.defined %}
+{%         if mpls.rsvp.refresh.interval is arista.avd.defined %}
+   refresh interval {{ mpls.rsvp.refresh.interval }}
+{%         endif %}
+{%         if mpls.rsvp.refresh.method is arista.avd.defined %}
+   refresh method {{ mpls.rsvp.refresh.method }}
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.hello is arista.avd.defined %}
+   hello interval {{ mpls.rsvp.hello.interval }} multiplier {{ mpls.rsvp.hello.multiplier }}
+{%     endif %}
+{%     if mpls.rsvp.authentication is arista.avd.defined %}
+{%         set auth = mpls.rsvp.authentication %}
+{%         if auth.type is arista.avd.defined %}
+   authentication type {{ auth.type }}
+{%         endif %}
+{%         if auth.sequence_number_window is arista.avd.defined %}
+   authentication sequence-number window {{ auth.sequence_number_window }}
+{%         endif %}
+{%         for password_index in auth.password_indexes | arista.avd.natural_sort('index') %}
+   authentication index {{ password_index.index }} password {{ password_index.password_type }} {{ password_index.password | arista.avd.hide_passwords(hide_passwords) }}
+{%         endfor %}
+{%         if auth.active_index is arista.avd.defined %}
+   authentication index {{ auth.active_index }} active
+{%         endif %}
+{%     endif %}
+{%     for neighbor in mpls.rsvp.neighbors | arista.avd.natural_sort('ip') %}
```

Comment:
can you check the sorting just base on ip address or it keep ipv4 address sorting separate from ipv6 in EOS

Reply:
Fixed!